spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/annotation/SqsListener.java

```diff
@@ -137,4 +137,10 @@
 	 */
 	String messageVisibilitySeconds() default "";
 
+	/**
+	 * The acknowledgement mode to be used for the provided queues. If not specified, the acknowledgement mode defined
+	 * for the container factory will be used.
+	 */
+	String acknowledgementMode() default SqsListenerAcknowledgementMode.DEFAULT;
```

Comment:
Now that we've settled with String, we don't need this DEFAULT anymore, we can't simply default to `""`.

Then if the string has text we resolve the value, otherwise we leave it as null.

How does that sound?

Reply:
It does make sense haha. Thanks!

Reply:
Ok, still missing this change.

Reply:
I was waiting for the feedback on the other comment to push the changes.. Sorry! It's now up to date